main.go

```diff
@@ -359,7 +394,20 @@ func getLinksInPage(url string) []string {
 					link := a.Val
 					matched, _ := regexp.MatchString("^http(s?)://", link)
 					if !matched {
-						link = url + link
+						tmpUrl := url
```

Comment:
Looking good
Can we move it to a new function - maybe "clean link url" or something similar
Also write some comments on this process to make it more clear

Reply:
Done.